### ISSUE TITLE
Java consul integration

### ISSUE THREAD

**Issue by sagarkrkv** [airavata-courses/spring17-API-Server#1]
Can someone use this library [https://github.com/OrbitzWorldwide/consul-client](https://github.com/OrbitzWorldwide/consul-client) to connect and register the api server with consul.

Assume that consul will be located on localhost at the default 8500 port. 

You need to implement the register and check service function shown.

```
Consul consul = Consul.builder().build(); 
AgentClient agentClient = consul.agentClient();

String serviceName = "API-Server";
String serviceId = "1";

agentClient.register(8080, 3L, serviceName, serviceId); 
//change 8080 to the port on which API-Server is running.

agentClient.pass(serviceId); // this function need to be called continuosly every 1 second while the api server is running

**Comment by sagarkrkv** [comment]
The consul version will be 0.7.*, use the corresponding client library

**Comment by sagarkrkv** [comment]
name the service "API-Server" and add a tag "urlprefix-API-Server/"

**Comment by supreeth90** [comment]
I will take this up. I am writing a python script which registers the "API-server" service with consul using python-consul.

**Comment by sagarkrkv** [comment]
Fixed this issue with commit ddb110f
